desktop/cmp/input/RadioInput.js

```diff
@@ -95,7 +120,15 @@ const cmp = hoistCmp.factory(
                 disabled: opt.disabled,
                 label: opt.label,
                 value: opt.value,
-                className: 'xh-radio-input-option'
+                className: 'xh-radio-input-option',
+                onFocus: (e) => {
+                    model.onFocus();
+                    opt.onFocus ? opt.onFocus(e) : null;
+                },
+                onBlur: (e) => {
+                    model.onBlur(e);
+                    opt.onFocus ? opt.onBlur(e) : null;
```

Comment:
Are these call backs intended to be a new API on the options (and buttons?).  Also, looks like a copy/paste error here.

Reply:
Fixed the copy/paste error.
The callbacks are not intended to be "new" on the button elements.  They've always been there, haven't they?
They're just necessary to make sure the button group model is notified that it has focus if one of its buttons is focused, same re blur, but a little more complicated with blur, since blurring one button does not mean the buttongroup is blurred, if another button is focused.